src/Notifier.php

```diff
@@ -51,6 +58,9 @@ public function __construct($opt = [])
                 return $this->rootDirectoryFilter($notice);
             });
         }
+
+        $handler = (isset($this->opt['httpClient']) ? $this->opt['httpClient'] : null);
```

Comment:
Thanks. We should stick to this code then, since we want to support more PHP versions.